internal/pkg/cli/storage_init.go

```diff
@@ -1266,12 +1263,16 @@ Resource names are injected into your containers as environment variables for ea
 		auroraFlagSet.AddFlag(cmd.Flags().Lookup(f))
 	}
 
+	othersFlagSet := pflag.NewFlagSet("Others", pflag.ContinueOnError)
+	othersFlagSet.AddFlag(cmd.Flags().Lookup(storageAddIngressFromFlag))
+
 	cmd.Annotations = map[string]string{
 		// The order of the sections we want to display.
-		"sections":          `Required,DynamoDB,Aurora Serverless`,
+		"sections":          `Required,DynamoDB,Aurora Serverless,Others`,
```

Comment:
nit: maybe?
```suggestion
		"sections":          `Required,DynamoDB,Aurora Serverless,Optional`,
```

Reply:
done!

also I added this commit after your approval: https://github.com/aws/copilot-cli/pull/4518/commits/1a80aa98f7ab3d88d25a23b6f8349adbdff4b00d 😰